util/opentelemetry-util-genai/src/opentelemetry/util/genai/span_utils.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 from dataclasses import asdict
-from typing import List
+from typing import Any, Dict, List, Optional
```

Comment:
nit prefer to use `dict[k, v]`, and `Foo | None` with `from __future__ import annotations`.

WDYT?